package.json

```diff
@@ -13,14 +13,16 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
-    "typescript": "^4.9.5",
```

Comment:
Why did you delete it?

Reply:
It was moved as a dev dependency at some point, is typescript considered a prod library? or dev? 🤔

Reply:
It's a dev dependency. I asked because I didn't see it added to `devDependencies`.